crates/stages/src/stages/sender_recovery.rs

```diff
@@ -93,7 +91,9 @@ impl<DB: Database> Stage<DB> for SenderRecoveryStage {
         for chunk in
             &tx_walker.chunks(self.commit_threshold as usize / rayon::current_num_threads())
         {
-            let tx = tx.clone();
+            // An _unordered_ channel to receive results from a rayon job
+            let (tx, rx) = mpsc::unbounded_channel();
+            channels.push(rx);
```

Comment:
oh right, now that we chunk it this is now way easier/better and already guarantees same ord.